lib/mix/tasks/sentry.send_test_event.ex

```diff
@@ -26,6 +26,8 @@ defmodule Mix.Tasks.Sentry.SendTestEvent do
     type: :string
   ]
 
+  @requirements ["app.start"]
+
```

Comment:
I don't like this Req-specific change inside the test command but I left it like this for now because I wanted to get some general feedback first.

Details why this is needed: https://elixirforum.com/t/argumenterror-unknown-registry-req-finch-on-a-simple-http-request-using-req-library/64638/9